lib/parse/duration/parsers/iso8601.ex

```diff
@@ -54,6 +60,8 @@ defmodule Timex.Parse.Duration.Parsers.ISO8601Parser do
             {true, Duration.add(d, Duration.from_minutes(m))}
           {?S, s}, {true, d} ->
             {true, Duration.add(d, Duration.from_seconds(s))}
+          {?W, _w}, {_, _} ->
+            {:error, "W cannot be used with other time period designators"}
```

Comment:
Perhaps the error will be more helpful if we phrase it like:

```elixir
{?W, _w}, {_, _} ->
  {:error, "Found 'W', a basic format designator, but the parse indicates " <>
               "this is an extended format, mixing the two formats is disallowed"}
```

Reply:
:+1: I've amended the commit